wagtaildraftail/client/wagtaildraftail.js

```diff
@@ -54,4 +64,18 @@ const initDraftailEditor = (fieldName, options = {}) => {
 
 window.initDraftailEditor = initDraftailEditor;
```

Comment:
@thibaudcolas: Do we want this to be `window.wagtailDraftail.initEditor` now that we have a namespace for other utils?